packages/edit-site/src/components/sidebar/style.scss

```diff
@@ -136,3 +136,10 @@ $split-border-control-offset: 55px;
 		}
 	}
 }
+
+// Override the `hr` styles defined for the complementary area component
+// from the `@wordpress/interface` package.
+.edit-site-global-styles-sidebar hr {
+	margin: 0;
+	border-color: rgba(0, 0, 0, 0.1);
```

Comment:
Why do we need to override the if it's mainly the margin that need zeroing out?

While diving in here, I also noticed that the previous color is wrong. It outputs #f0f0f0 (which maps to $gray-100), but it should have been #e0e0e0 ($gray-200) like the other panel borders.

Depending on your thoughts here (maybe there's a reason you used opacity), I could imagine two paths forward:

1. We remove the border-color override here, and accept the wrong color coming from the component directly, and then update the component to use the correct color (#e0e0e0) in a separate PR.
2. We remove he border-color override here, _and_ fix the component in the same PR. 

What do you think?

Reply:
> maybe there's a reason you used opacity

The separator in the Global Styles sidebar is rendered as a `CardDivider`, and the color of this component is `rgba(0, 0, 0, 0.1)` (as [per the config](https://github.com/WordPress/gutenberg/blob/e4252e430a04e0f528142a5a1f56e2b1971ecfce/packages/components/src/utils/config-values.js#L43)).

The style overrides introduced by `ComplementaryArea` [changed the separator color to `$gray-100`](https://github.com/WordPress/gutenberg/blob/c1ea8d60db9f05ebbfbaced50a88bfe30a7a0270/packages/interface/src/components/complementary-area/style.scss#L55), and therefore I introduced this rule to bring the separator color back to how the original component was intended to be.

> I could imagine two paths forward:

I will go ahead and drop this color override for the time being.

As the next steps, there will be a few separate fronts to work on:

**1. Review separator and border colors of `@wordpress/components`**

We can definitely look at reviewing and updating the color used by components in `@wordpress/components` (e.g. [card](https://github.com/WordPress/gutenberg/blob/e4252e430a04e0f528142a5a1f56e2b1971ecfce/packages/components/src/utils/config-values.js#L43), [surface](https://github.com/WordPress/gutenberg/blob/e4252e430a04e0f528142a5a1f56e2b1971ecfce/packages/components/src/utils/config-values.js#L76), [datepicker](https://github.com/WordPress/gutenberg/blob/c3ff86d88f35ceb92bfc2ad58a335fc9de7e9e09/packages/components/src/date-time/datepicker.scss#L513)...)

Although I wasn't involved in the conversation at the time, if I had to guess  that color is currently using transparency to make it blend better in case the background color is not white

**2. Review the `rgba(0, 0, 0, 0.1)` color across Gutenberg**

Depending on the results of the previous step, we should also review the remaining usage of `rgba(0, 0, 0, 0.1)` in the [color panel](https://github.com/WordPress/gutenberg/blob/8675f86f885ca389f3bca39971b22cb897bab61b/packages/block-editor/src/hooks/color.scss#L34-L42)

**3. Review the style overrides in `ComplementaryArea`**

Despite the changes from the points above, the separator in the Global Styles sidebar will keep using `$gray-100` because of the [style overrides from `ComplementaryArea`](https://github.com/WordPress/gutenberg/blob/c1ea8d60db9f05ebbfbaced50a88bfe30a7a0270/packages/interface/src/components/complementary-area/style.scss#L55). We could start by changing the color used by these style overrides, or we could work on a more structured solution (see https://github.com/WordPress/gutenberg/pull/40526#discussion_r855973432).

Reply:
Ah, excellent context. It does sound like we'll want to fix that at the component level.

At the _moment_ we have these shades of gray for most of the UI:

```
$gray-900: #1e1e1e;
$gray-800: #2f2f2f;
$gray-700: #757575;		// Meets 4.6:1 text contrast against white.
$gray-600: #949494;		// Meets 3:1 UI or large text contrast against white.
$gray-400: #ccc;
$gray-300: #ddd;		// Used for most borders.
$gray-200: #e0e0e0;		// Used sparingly for light borders.
$gray-100: #f0f0f0;		// Used for light gray backgrounds.
```

In this case, the 0.1 opacity divider actually matches `$gray-100`, but it ideally _should_ match `$gray-200` as those are used on panels and general UI.

I understand there are challenges with making the components work on a dark background, and that it can therefore make sense to use more agnostic colors that aren't optimized for a white background, perhaps even opacities. I trust that'll be taken into account 👍 👍